src/app/app.component.html

```diff
@@ -1 +1,7 @@
-<app-header></app-header>
\ No newline at end of file
+
+  <div class="header">
+    <app-header></app-header>
+  </div>
+  <div class="footer">
+    <app-footer></app-footer>
+  </div>
```

Comment:
Why do you need divs?
```suggestion
<app-header></app-header>
<app-footer></app-footer>
```